client/my-sites/customize/controller.js

```diff
@@ -1,41 +1,40 @@
 /**
  * External Dependencies
  */
-var i18n = require( 'i18n-calypso' ),
-	ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	ReduxProvider = require( 'react-redux' ).Provider;
+import i18n from 'i18n-calypso';
+import ReactDom from 'react-dom';
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
 
 /**
  * Internal Dependencies
  */
-var sites = require( 'lib/sites-list' )(),
-	route = require( 'lib/route' ),
-	analytics = require( 'lib/analytics' ),
-	setTitle = require( 'state/document-head/actions' ).setDocumentHeadTitle;
+import sitesFactory from 'lib/sites-list';
+import { sectionify } from 'lib/route/path';
+import analytics from 'lib/analytics';
+import { setDocumentHeadTitle as setTitle } from 'state/document-head/actions';
 
-module.exports = {
+const sites = sitesFactory();
 
-	customize: function( context ) {
-		var CustomizeComponent = require( 'my-sites/customize/main' ),
-			basePath = route.sectionify( context.path );
+export function customize( context ) {
+	const CustomizeComponent = require( 'my-sites/customize/main' ),
+		basePath = sectionify( context.path );
 
-		analytics.pageView.record( basePath, 'Customizer' );
+	analytics.pageView.record( basePath, 'Customizer' );
 
-		context.store.dispatch( setTitle( i18n.translate( 'Customizer', { textOnly: true } ) ) ); // FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+	// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+	context.store.dispatch( setTitle( i18n.translate( 'Customizer', { textOnly: true } ) ) );
 
-		ReactDom.render(
-			React.createElement( ReduxProvider, { store: context.store },
-				React.createElement( CustomizeComponent, {
-					domain: context.params.domain || '',
-					sites: sites,
-					prevPath: context.prevPath || '',
-					query: context.query,
-					panel: context.params.panel
-				} )
-			),
-			document.getElementById( 'primary' )
-		);
-	}
-
-};
+	ReactDom.render(
```

Comment:
Drive-by, but what about using one of the helpers, which may be more conducive to single-tree-codemodding?